src/components/helpers/form-helpers/parseEventDate.js

```diff
@@ -0,0 +1,7 @@
+import moment from "moment";
```

Comment:
Rename file to dateParser, so it can contain multiple functions related to date handling

Reply:
Done